dataflow/java/src/main/java/com/httparchive/dataflow/BigQueryImport.java

```diff
@@ -125,7 +141,7 @@ public static Response truncateUTF8(String s, int maxBytes) {
                             more += 3 + t.length();
                         }
                 }
-
+                
```

Comment:
Oh gross I didn't know it was doing that. I'm using Sublime today on my laptop and NetBeans on the workstation. I'll see if there's a setting to clean that up.